nautobot/dcim/tests/test_views.py

```diff
@@ -1275,10 +1277,11 @@ def setUpTestData(cls):
         }
 
         cls.csv_data = (
-            "role,manufacturer,device_type,status,name,location,rack_group,rack,position,face,secrets_group",
-            f"{deviceroles[0].name},{manufacturer.name},Device Type 1,{statuses[0].name},Device 4,{locations[0].name},Rack Group 1,Rack 1,10,front,",
-            f"{deviceroles[0].name},{manufacturer.name},Device Type 1,{statuses[0].name},Device 5,{locations[0].name},Rack Group 1,Rack 1,20,front,",
-            f"{deviceroles[0].name},{manufacturer.name},Device Type 1,{statuses[0].name},Device 6,{locations[0].name},Rack Group 1,Rack 1,30,front,Secrets Group 2",
+            "role,device_type,status,name,location,rack,position,face,secrets_group,parent_bay",
```

Comment:
Right, in the old CSV form it was used as a way to narrow down which device type was being selected; since we're using natural-keys now they serve the same purpose without needing to include non-direct fields as separate columns.